inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java

```diff
@@ -353,44 +353,4 @@ private void checkParams(SinkRequest request) {
         Preconditions.checkNotNull(sinkName, ErrorCodeEnum.SINK_NAME_IS_NULL.getMessage());
     }
 
-    /**
-     * Asynchronously initiate a single inlong stream related workflow
```

Comment:
Please do not remove those codes, because we will recover them later.

Reply:
OK